packages/forms-web-app/src/views/interested-party-guide/have-your-say-examination.njk

```diff
@@ -0,0 +1,99 @@
+{% extends "layouts/layout.njk" %}
+
+{% block pageTitle %}
+  Have your say during the examination of the application
+{% endblock %}
+
+{% block header %}
+  {{ pinsHeader() }}
+{% endblock %}
+
+{% block beforeContent %}
+
+  {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "alpha"
+    },
+    html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+  }) }}
+
+{% endblock %}
+{% block content %}
+
+  <h1 class="govuk-heading-xl">
+        Have your say during the examination of the project
+    </h1>
+  <h2 class="pins-contents-list-title govuk-heading-s">Contents</h2>
+  <ol class="govuk-list pins-contents-list">
+    <li>
+      <a class="govuk-link" href="#what-examination-means">What examination means</a>
+    </li>
+    <li>
+      <a class="govuk-link" href="#how-to-comment-during-examination">How to comment during examination</a>
+    </li>
+    <li>
+      <a class="govuk-link" href="#missed-deadline">If you have missed the deadline to register</a>
+    </li>
+    <li>
+      <a class="govuk-link" href="#more-detailed-advice">More detailed advice</a>
+    </li>
+  </ol>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="pins-step-by-step-section-main">
+
+        <h2 id="what-examination-means" class="govuk-heading-m">What examination means</h2>
```

Comment:
Should be What Examination means ( with capital E )

Reply:
Fixed